src/components/Create/components/pages/PayoutsPage/OldPayoutsPage.tsx

```diff
@@ -0,0 +1,293 @@
+import { StopOutlined } from '@ant-design/icons'
```

Comment:
Absolutely nothing changed in this file from when it was the OG `PayoutsPage.tsx`

Reply:
Heads up: pushing a commit to reverse the naming of these. When we clean up feature flag, we can rename `NewPayoutsPage` to `PayoutsPage`